x-pack/plugins/cases/public/components/visualizations/actions/action_wrapper.tsx

```diff
@@ -30,16 +29,15 @@ const ActionWrapperWithContext: React.FC<PropsWithChildren<Props>> = ({
   caseContextProps,
   currentAppId,
 }) => {
-  const { application } = useKibana().services;
-  const isDarkTheme = useIsDarkTheme();
```

Comment:
`useIsDarkTheme` seems to just be passing through the dark theme setting from core, which isn't explicitly needed with `KibanaRenderContextProvider`